### PR TITLE
KMS-9186 - Fix WYSIWYG Token Replacement Code

### PR DESCRIPTION
**Pull request by muli**

### CODE REVIEW

**Comment by muli** [comment]
@gonenradai, can you please review?
Please note that although it looks like they are a lot of changes in the JavaScript files, it is mainly due to whitespaces... Not actual changes.

File: lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js [hunk 1]
```diff
@@ -120,7 +120,17 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
         
         return courseId;
     },
-            
+
+    _replaceKafUriWithToken: function(sourceUrl, kafUri, kalturaToken) {
+        sourceUrl = this._removeProtocolFromUrl(sourceUrl);
+        kafUri = this._removeProtocolFromUrl(kafUri);
+        return sourceUrl.replace(kafUri, kalturaToken);
```
Comment (gonenradai) [inline lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js:127]: isn't the result here a URL without schema?
if I'm not mistaken - we used to save the schema as part of the embed before this change. not sure it is smart to start storing URLs without schema.

Comment (muli) [inline lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js:127]: @gonenradai - the result of this particular function indeed returns a URL without a schema, however, the schema is being added to the page later on (line 158).

This is the WYSIWYG fix, there will be another PR which fixes the way we actually save the source URL in the DB, but with the correct path.

Comment (gonenradai) [inline lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js:127]: why not a single PR that includes the DB-insert as well?

as for the removal of the http and it being added later - OK.

**Comment by muli** [comment]
@gonenradai, please see PR #59 - I messed up this branch & PR too much so I created a new one.

**Pull request closed without merging**